### PR TITLE
Add manifest for JOLTS

### PR DESCRIPTION
**Pull request by intrepiditee**
`0 10 17 * *` means `At 10:00 on day-of-month 17`, according to [https://crontab.guru/#0_10_17_*_*](https://crontab.guru/#0_10_17_*_*).

This schedule is chosen because according to [https://www.bls.gov/schedule/2020/home.htm](https://www.bls.gov/schedule/2020/home.htm), Job Openings and Labor Turnover Survey comes out at the latest on 17th every month.

Thanks for reviewing.

### CODE REVIEW

File: scripts/us_bls/jolts/manifest.json [hunk 1]
```diff
@@ -0,0 +1,23 @@
+{
+    "import_specifications": [
+        {
+            "import_name": "BLS_JOLTS",
+            "curator_emails": [
+                "ianjc@google.com"
```
Comment (jeffreyoldham) [inline scripts/us_bls/jolts/manifest.json:6]: \# TODO(jeffreyoldham): Update to another maintainer when appropriate.

File: scripts/us_bls/jolts/manifest.json [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+                "bls_jolts.py"
+            ],
+            "import_inputs": [
+                {
+                    "template_mcf": "BLSJolts.tmcf",
+                    "cleaned_csv": "BLSJolts.csv",
+                    "node_mcf": "BLSJolts_StatisticalVariables.mcf"
+                }
+            ],
+            "cron_schedule": "0 10 17 * *"
```
Comment (jeffreyoldham) [inline scripts/us_bls/jolts/manifest.json:20]: Did we choose this to match the typical release dates?

Comment (intrepiditee) [reply scripts/us_bls/jolts/manifest.json:20]: It seems like there isn't a set release date. We can also make it the last day of a month

Comment (jeffreyoldham) [reply scripts/us_bls/jolts/manifest.json:20]: @IanCostello I defer to Ian's opinion.

Comment (IanCostello) [reply scripts/us_bls/jolts/manifest.json:20]: It seems to always come out by the 19th if I recall. By the end of the month could be a good option. Or on the 21st and the 31st to be safe.

Comment (intrepiditee) [reply scripts/us_bls/jolts/manifest.json]: changed to 21st

**Pull request merged**